src/webui/src/components/logPath/PaiTrialLog.tsx

```diff
@@ -44,7 +46,7 @@ class PaitrialLog extends React.Component<PaitrialLogProps, {}> {
                                     </a>
                                     <a target="_blank" href={logStr.split(',')[1]}>hdfsLog</a>
                                 </Row>
-                                <Row>
+                                {/* <Row>
```

Comment:
We can delete these code lines, rather than just comment out

Reply:
ok